Makefile

```diff
@@ -5,117 +5,57 @@
 # container images of the osbuild-containers repository.
 #
 
-#
-# Global Setup
-#
-# This section sets some global parameters that get rid of some old `make`
-# annoyences.
-#
-#     SHELL
-#         We standardize on `bash` for better inline scripting capabilities,
-#         and we always enable `pipefail`, to make sure individual failures
-#         in a pipeline will be treated as failure.
-#
-#     .SECONDARY:
-#         An empty SECONDARY target signals gnu-make to keep every intermediate
-#         files around, even on failure. We want intermediates to stay around
-#         so we get better caching behavior.
-#
-
-SHELL			:= /bin/bash -eox pipefail
-
-.SECONDARY:
-
-#
-# Parameters
-#
-# The set of global parameters that can be controlled by the caller and the
-# calling environment.
-#
-#     BUILDDIR
-#         Path to the directory used to store build artifacts. This defaults
-#         to `./build`, so all artifacts are stored in a subdirectory that can
-#         be easily cleaned.
-#
-#     SRCDIR
-#         Path to the source code directory. This defaults to `.`, so it
-#         expects `make` to be called from within the source directory.
-#
-#     BIN_*
-#         For all binaries that are executed as part of this makefile, a
-#         variable called `BIN_<exe>` defines the path or name of the
-#         executable. By default, they are set to the name of the binary.
-#
+BIN_DOCKER ?= docker
+BIN_JQ     ?= jq
+BIN_MKDIR  ?= mkdir
 
-BUILDDIR		?= ./build
-SRCDIR			?= .
+IMG_BAKE_ARGS ?=
+IMG_BUILDER   ?= default
 
-BIN_DOCKER		?= docker
-BIN_JQ			?= jq
-BIN_MKDIR		?= mkdir
-
-#
-# Generic Targets
-#
-# The following is a set of generic targets used across the makefile. The
-# following targets are defined:
-#
-#     help
-#         This target prints all supported targets. It is meant as
-#         documentation of targets we support and might use outside of this
-#         repository.
-#         This is also the default target.
-#
-#     $(BUILDDIR)/
-#     $(BUILDDIR)/%/
-#         This target simply creates the specified directory. It is limited to
-#         the build-dir as a safety measure. Note that this requires you to use
-#         a trailing slash after the directory to not mix it up with regular
-#         files. Lastly, you mostly want this as order-only dependency, since
-#         timestamps on directories do not affect their content.
-#
-#     FORCE
-#         Dummy target to use as dependency to force `.PHONY` behavior on
-#         targets that cannot use `.PHONY`.
-#
+# Get acces to extra command line arguments
+MAKE_ARGS := $(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+# Make sure they will not be treated as active targets
+$(eval $(MAKE_ARGS):;@:)
```

Comment:
TBH, I find this a bit cryptic.

I propose to keep the `IMG_TARGET` defined as it was. With your changes, it won't have any default value specified if not provided. One can still override the `IMG_TARGET` by explicitly passing it to make as

```sh
make bake IMG_TARGET=osbuild-ci-latest
```